pkg/providers/instancetype/skus_test.go

```diff
@@ -16,21 +16,21 @@ limitations under the License.
 
 package instancetype
 
-import "k8s.io/apimachinery/pkg/util/sets"
-
-var (
```

Comment:
FYI this is a weird diff, the `RestrictedVMSizes` var has actually been moved to the newly added skus.go file.